userprofile/api/views.py

```diff
@@ -12,6 +12,13 @@
 from .full_serializers import *
 
 
+class CustomSearchFilter(filters.SearchFilter):
```

Comment:
This filter is related to the initial_selection AJAX in aplus.js. This is useless in this format since single user details can be fetched from the API without any search. If we could fetch multiple given users, then this might be changed.